### PR TITLE
Small UI Design Improvements

### PR DESCRIPTION
**Pull request by Parveshdhull**
# Small UI Design Improvements

#### Summary
Fixed distortion of Status Bar due to opening of Bottom Sheet (In Android Device)
<div align="center">
Screenshot
</div>

#### Summary
Fixed cropping of Subtitle texts

<div align="center">
Screenshot
   <div width="40"/>
Screenshot
</div>

### CODE REVIEW

File: src/quo/components/list/item.cljs [hunk 1]
```diff
@@ -198,7 +198,7 @@
            title subtitle subtitle-secondary active on-press on-long-press chevron size text-size
            accessory-text accessibility-label title-accessibility-label accessory-style
            haptic-feedback haptic-type error animated animated-accessory? title-text-weight]
-    :or   {subtitle-max-lines 1
+    :or   {subtitle-max-lines 0
```
Comment (flexsurfer) [inline src/quo/components/list/item.cljs]: I'm not sure we want to change the default value for the component, better to pass a parameter for this case only

**Review by flexsurfer — CHANGES_REQUESTED** [review]

**Comment by Parveshdhull** [comment]
Hi @flexsurfer ,
I restored the default value and passed the parameter to all 4 instances where overflowing was happening.

File: src/quo/components/list/item.cljs (hunk 1 above)
Comment (Parveshdhull) [reply src/quo/components/list/item.cljs]: Hi @flexsurfer ,
I restored the default value and passed the parameter to 4 instances where overflowing was happening.

**Review by flexsurfer — APPROVED** [review]

**Comment by qoqobolo** [comment]
@Parveshdhull thanks for these improvements, looks better now!
PR is tested and can be merged.

**Pull request merged**